README.md

```diff
@@ -178,212 +174,15 @@ yarn build:linux
 ```
 
 ## Contributors
+Made with [contrib.rocks](https://contrib.rocks).
+
+> [!NOTE]
+> It may take up to a day for the list to be updated.
```

Comment:
I don't think this note is needed as well

Reply:
I mean, I know there is a delay for it to update. But that is no problem for us

Reply:
only the note or also the "made with..." part?

Reply:
removed "made with..." part because it is a pain to translate it to all languages and it is referenced on the url image anyway heheh